migrations/postgres/20241029052632_values.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+-- create "values" table
+CREATE TABLE "values" (
+  "env_id" uuid NOT NULL,
```

Comment:
I think we should add created_at and updated_at columns

We should also think about deleted_at for soft delete
Cause users might delete this value while a workflow is still using them and can replay

Reply:
adding these for debugging/auditing reasons makes sense, but it doesn't matter for replayability - these things will run in activities which will save the result and will not run multiple times.

Reply:
added updated_at